### ISSUE TITLE
 Cannot use apksigner tool for signature verification. Original error: JAVA_HOME is not set currently. Please set JAVA_HOME.

### ISSUE THREAD

**Issue by AlpBilgin** [appium/appium#14410]
## The problem

I updated to 1.17.1 and tests on Android simulator got broken.

## Environment

* Appium version (or git revision) that exhibits the issue: 1.17.1
* Last Appium version that did not exhibit the issue (if applicable):1.15.1
* Desktop OS/version used to run Appium: MacOS 10.15.5
* Node.js version (unless using Appium.app|exe): 11.4.0
* Npm or Yarn package manager: npm 6.10.3
* Mobile platform/version under test: android 9
* Real device or emulator/simulator: emulator
* Appium CLI or Appium.app|exe: CLI

## Details

I reinstalled latest appium through npm and tried the test suite on an Android Emulator target. It stopped working.  JAVA_HOME is set and Java executable is under it

 echo $JAVA_HOME yields /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home

## Link to Appium logs

https://gist.github.com/AlpBilgin/e2d65f6d85920a7665697793b310a1a2

## Code To Reproduce Issue [ Good To Have ]

Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

Please git clone https://github.com/appium/appium and from the `sample-code` directory, use one of your favourite languages and sample apps to reproduce the issue.

In case a similar scenario is missing in sample-code, please submit a PR with one of the sample apps provided.

**Comment by mykola-mokhnach** [comment]
Yes this is a duplicate. Please follow https://github.com/appium/appium/issues/14408#issuecomment-637946417
